modules/nf-core/mindagap/mindagap/main.nf

```diff
@@ -0,0 +1,33 @@
+process MINDAGAP {
```

Comment:
As the folder structure is TOOL/SUBTOOL the name should match this. 
```suggestion
process MINDAGAP_MINDAGAP {
```
Same in other places such as `meta.yml` or test files.